src/libusermode/running.cpp

```diff
@@ -394,7 +394,7 @@ event_response_t wait_for_target_process_cb(
 {
     rh_data_t* rh_data = static_cast<rh_data_t*>(info->trap->data);
     // Wait for target_process.
-    if (info->proc_data.pid != rh_data->target_process_pid)
+    if (info->attached_proc_data.pid != rh_data->target_process_pid)
```

Comment:
I think there needs to be a check to verify that the target is Windows for this change. Even if `libusermode` is only for Windows right now in the future it should be usable on Linux too.

Reply:
Done.